lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerLaunchTests.cs

```diff
@@ -38,6 +38,19 @@ public class PuppeteerLaunchTests : PuppeteerBaseTest
             }
         }
 
+        [Fact]
+        public async Task ShouldWorkInRealLifeWithOptions()
+        {
+            var options = TestConstants.DefaultBrowserOptions();
+
+            using (var browser = await PuppeteerSharp.Puppeteer.LaunchAsync(options, TestConstants.ChromiumRevision))
+            using (var page = await browser.NewPageAsync())
+            {
+                var response = await page.GoToAsync("https://www.google.com", new NavigationOptions() { Timeout = 5000, WaitUntil = new[] { "networkidle0" } });
```

Comment:
Can we make this multi-line?

```cs 
var response = await page.GoToAsync("https://www.google.com", new NavigationOptions
{ 
    Timeout = 5000, 
    WaitUntil = new[] { "networkidle0" }
});
```

Reply:
Stylecop recommends putting each parameter on its own line in that case, is that ok?

```csharp
                var response = await page.GoToAsync(
                    "https://www.google.com",
                    new NavigationOptions()
                    {
                        Timeout = 5000,
                        WaitUntil = new[] { "networkidle0" }
                    });
```